src/gtest/test_mnode_rpc.cpp

```diff
@@ -86,3 +90,30 @@ INSTANTIATE_TEST_SUITE_P(mnode_rpc_ani2psl, PTest_ani2psl_secret,
         }, true, true)
 	));
 
+class PTest_ASCII85_Encode_Decode: public TestWithParam<tuple<string, string>>       
+{
+public:
+    static void SetUpTestCase()
+    {
+        SelectParams(CBaseChainParams::Network::REGTEST);
+    }
+};
+
+
+TEST_P(PTest_ASCII85_Encode_Decode, ASCII85_Encode_Decode) {
+
+    const string to_be_encoded_1 = get<0>(GetParam()); //"hello"; // Encoded shall be : "BOu!rDZ"
```

Comment:
Yes, thank you. It is much easier to add new special test cases to parameterized tests.
Especially for any encode/decode, encrypt/decrypt methods testing.